xbmc/utils/BitstreamConverter.h

```diff
@@ -17,6 +17,10 @@ extern "C" {
 #include <libavformat/avformat.h>
 #include <libavfilter/avfilter.h>
 #include <libavcodec/avcodec.h>
+
+#ifdef HAVE_LIBDOVI
+#include <libdovi/rpu_parser.h>
```

Comment:
Can you forward declare `DoviData` here instead? Then we don't need the ifdef

Reply:
I'm confused if it's still `class` for this
```c
typedef struct {
  ...
} DoviData;
```

Reply:
Ok, thanks.